samples/templates/express-embedded-auth-with-sdk/web-server/server.js

```diff
@@ -18,6 +18,7 @@ const express = require('express');
 const session = require('express-session');
 const mustacheExpress = require('mustache-express');
 const path = require('path');
+require('express-async-errors');
```

Comment:
due to v6 change, idx errors is thrown directly now, add this middleware for error handling purpose